### ISSUE TITLE
What do I put in the customSearch.json file?

### ISSUE THREAD

**Issue by DragonZV** [Cramenorn/RaptorSA#19]
Oh yea also this 

```internal/modules/cjs/loader.js:895
  throw err;
  ^

Error: Cannot find module 'C:\Users\akyra\Desktop\Bots\RaptorSA\libs/../../config/googleResults.json'
Require stack:
- C:\Users\akyra\Desktop\Bots\RaptorSA\libs\google.js
- C:\Users\akyra\Desktop\Bots\RaptorSA\src\commands\randomstuff\google.js
- C:\Users\akyra\Desktop\Bots\RaptorSA\src\index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:892:15)
    at Function.Module._load (internal/modules/cjs/loader.js:742:27)
    at Module.require (internal/modules/cjs/loader.js:964:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (C:\Users\akyra\Desktop\Bots\RaptorSA\libs\google.js:9:27)
    at Module._compile (internal/modules/cjs/loader.js:1075:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1096:10)
    at Module.load (internal/modules/cjs/loader.js:940:32)
    at Function.Module._load (internal/modules/cjs/loader.js:781:14)
    at Module.require (internal/modules/cjs/loader.js:964:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\akyra\\Desktop\\Bots\\RaptorSA\\libs\\google.js',
    'C:\\Users\\akyra\\Desktop\\Bots\\RaptorSA\\src\\commands\\randomstuff\\google.js',
    'C:\\Users\\akyra\\Desktop\\Bots\\RaptorSA\\src\\index.js'
  ]
}

**Comment by MalwareWerewolf** [comment]
Hi @Dragonizedpizza ,
this issue has been corrected with the commit [e33b2cd](https://github.com/Cramenorn/RaptorSA/commit/e33b2cd719710819ede6a90b275a4500a0a29925).

**Comment by MalwareWerewolf** [comment]
**customSearch.json** saves the queries made by Google APIs to read them in a second moment in order to display them on a Discord channel.
